src/utils/timeFromNow.jsx

```diff
@@ -0,0 +1,127 @@
+import ClassNames from 'classnames';
+import moment from 'moment-timezone';
+import PropTypes from 'prop-types';
+import React from 'react';
+
+const propTypes = {
+    className: PropTypes.string,
+    date: PropTypes.oneOfType([
+        PropTypes.string,
+        PropTypes.shape({}),
```

Comment:
I'm assuming it's really "one of type: string or moment object" ... right?  I'm wondering if we should consider a custom check function to verify that it is in fact a moment object ( using [`moment.isMoment()`](https://momentjs.com/docs/#/query/is-a-moment/) ), similar to [what I did for the `<DurationPicker />` component](https://github.com/saddlebackdev/react-cm-ui/blob/dev/src/molecules/durationPicker.js#L293-L301).  At the time I wrote the `<DurationPicker />` I did discover there is a [`react-moment-proptypes`](https://www.npmjs.com/package/react-moment-proptypes) package that has these built in that we can consider.

All of this is firmly in NAB territory of course.  Up to you if you want to code the custom prop type check function for a moment object or not.